wellregistry/registry/models.py

```diff
@@ -232,6 +233,42 @@ class MonitoringLocation(models.Model):
     class Meta:
         unique_together = (('site_no', 'agency'),)
 
+    def clean(self):
```

Comment:
This code was effectively in the admin module. Moving the code into the model clean method, allows it to be used for validation of both the admin editing forms as well as the new bulk upload feature.